frontend/src/routes/ProtectedRoute.tsx

```diff
@@ -38,7 +38,9 @@ export const AuthorizedRoute = ({
       allowedRoles.includes(currentUserRole) || isPMOSysAdmin ? (
         <Outlet />
       ) : (
-        <Navigate to={`/unauthorized`} />
+        // TODO Bug fix need for unauthorized
```

Comment:
did you mean to leave this in... i would revert it back, as this is not the fix.